src/overlay.rs

```diff
@@ -107,13 +108,13 @@ macro_rules! screen_print {
         $crate::screen_print!(@impl push, sec: $timeout, col: None, $text $(, $fmt_args)*);
```

Comment:
This one needs type too.
```suggestion
        $crate::screen_print!(@impl push, sec: $timeout, col: Option::<Srgba>::None, $text $(, $fmt_args)*);
```
Or `None::<Srgba>`.